lua/entities/gmod_tardis/modules/cl_music.lua

```diff
@@ -22,10 +22,12 @@ TARDIS:AddSetting({
 })
 
 function ENT:StopMusic()
-	if IsValid(self.music) then
+    if IsValid(self.music) then
+        if self.music:GetState() == GMOD_CHANNEL_PLAYING then
+			TARDIS:Message(LocalPlayer(), "Music stopped.")
```

Comment:
Messages don't tend to use full stops at the end

```suggestion
			TARDIS:Message(LocalPlayer(), "Music stopped")
```